### ISSUE TITLE
bug: `asdf local` warns about "tails" of no such file or directory

### ISSUE THREAD

**Issue by x80486** [asdf-vm/asdf#1425]
### Describe the Bug

Not sure what's going on here, but I've been seeing this for a couple of days now. This is something I don't set frequently, but the error/warning is happening for all `asdf local ...` commands:

```shell
[x80486@uplink:~/Workshop/struts-sandbox]$ asdf local java temurin-8.0.352+8 
tail: cannot open '/home/x80486/Workshop/struts-sandbox/.tool-versions' for reading: No such file or directory
```

### Steps to Reproduce

1. Install `asdf` version manager
2. Install a plugin and any version
3. Setup a global version — not sure if this is relevant
4. Setup a local version

### Expected Behaviour

No errors/warnings should occur and the local version along with the `.tool-versions` file should be created and set correctly.

### Actual Behaviour

An error/warning in the CLI

### Environment

```shell
[x80486@uplink:~/Workshop/struts-sandbox]$ asdf info 
OS:
Linux uplink 5.15.86-1-lts #1 SMP Sun, 01 Jan 2023 16:03:00 +0000 x86_64 GNU/Linux

SHELL:
GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

ASDF VERSION:
v0.11.0-6a4f51a

ASDF ENVIRONMENT VARIABLES:
ASDF_DIR=/home/x80486/.asdf

ASDF INSTALLED PLUGINS:
deno                         https://github.com/asdf-community/asdf-deno.git master 87442fe
golang                       https://github.com/kennyp/asdf-golang.git master f006a12
golangci-lint                https://github.com/hypnoglow/asdf-golangci-lint.git master a43c37c
gradle                       https://github.com/rfrancis/asdf-gradle.git master e47df00
java                         https://github.com/halcyon/asdf-java.git master 7026649
maven                        https://github.com/halcyon/asdf-maven.git master add7c1c
nodejs                       https://github.com/asdf-vm/asdf-nodejs.git master c9e5df4
terraform                    https://github.com/asdf-community/asdf-hashicorp.git master c44c314
v                            https://github.com/jthegedus/asdf-v.git main 99136a7
```

### asdf plugins affected (if relevant)

N/A

**Comment by Stratus3D** [comment]
Identical to https://github.com/asdf-vm/asdf/issues/1417.

@jthegedus what do you think about me getting 0.11.1 released this week? It would include the fix for this.

**Comment by jthegedus** [comment]
> Identical to #1417.
> 
> @jthegedus what do you think about me getting 0.11.1 released this week? It would include the fix for this.

Happy for this to be released. The general strategy for merging the release PR would be if you're comfortable with the state of HEAD. I use HEAD unless testing specific PRs, so I am happy with its stability.

**Comment by Stratus3D** [comment]
I'll test HEAD today and if everything goes well I'll merge 0.11.1 tomorrow morning.

**Comment by hyperupcall** [comment]
I think this should probably be closed now as both HEAD and 0.11.1 fix this issue

**Comment by jthegedus** [comment]
Closed by #1418
